### PR TITLE
Dataset species auto populate

### PR DESCRIPTION
**Pull request by temi**

### CODE REVIEW

**Review by chrisala — CHANGES_REQUESTED** [review]
Hi Temi, all the changes look good to me, however can you please add unit tests for the new ProjectController/ProjectService methods.

Also, while I'm not against the ALA harvest flag change it hasn't been discussed with DCCEEW - is it necessary for the value to be "Yes" for this to work?

File: src/main/scripts/releases/3.2/insertParatooProtocolConfig.js [hunk 1]
```diff
@@ -0,0 +1,451 @@
...
+        "usesPlotLayout": true,
+        "tags": ["survey"],
+        "apiEndpoint": "photopoints-surveys",
+    },
+    "383fa013-c52d-4186-911b-35e9b2375653": {
+        "name": "Photopoints - Compact Panorama",
+        "usesPlotLayout": true,
+        "tags": ["survey"],
+        "apiEndpoint": "photopoints-surveys",
+        "startDatePath": "attributes.start_date_time",
```
Comment (chrisala) [inline src/main/scripts/releases/3.2/insertParatooProtocolConfig.js:72]: The "attributes" prefix is no longer valid - I should have edited this when I made my change to get the dates and site from the /reverse-lookup call.

Comment (temi) [reply src/main/scripts/releases/3.2/insertParatooProtocolConfig.js]: fixed

**Comment by temi** [comment]
@chrisala Added logic to hide ala harvest feature

**Pull request merged**